fili-core/src/test/groovy/com/yahoo/bard/webservice/web/handlers/AsyncWebServiceRequestHandlerSpec.groovy

```diff
@@ -42,6 +42,7 @@ class AsyncWebServiceRequestHandlerSpec extends Specification {
         1 * dws.postDruidQuery(rc, _, null, null, groupByQuery) >> { a0, a1, a2, a3, a4 ->
             // Save the success callback
             sc = a1
+            return null
```

Comment:
If just `sc = a1` was failing, that means that it wasn't returning whatever it needed to (possibly because assignment doesn't have a return value). How was it failing with only `sc = a1`?